tests/visitors/test_models.py

```diff
@@ -1,9 +1,221 @@
+                    ],
+                )
+        """
+        self.assertCodemod(before, after)
+
+    # testing our on_delete detector works as expected in
+    # all the little edge cases
+    def test_multiple_kwargs_with_on_delete(self) -> None:
+        before = """
+            from django.db import models
+
+            class MyModel(models.Model):
+                operations = [
+                    migrations.CreateModel(
+                        fields=[
+                            (
+                                'model',
+                                models.OneToOneField(
+                                    to = 'random.Model', on_delete = models.CASCADE),
+                            ),
+                        ],
+                    )
+                ]
+        """
+        after = """
+            from django.db import models
+
+            class MyModel(models.Model):
+                operations = [
+                    migrations.CreateModel(
+                        fields=[
+                            (
+                                'model',
+                                models.OneToOneField(
+                                    to = 'random.Model', on_delete = models.CASCADE),
+                            ),
+                        ],
+                    )
+                ]
+        """
+        self.assertCodemod(before, after)
+
+    def test_multiple_kwargs_without_on_delete(self) -> None:
+        before = """
+            from django.db import models
+
+
+            class MyModel(models.Model):
+                operations = [
+                    migrations.AddField(
+                        field=models.ForeignKey(
+                            related_name='model',
+                            blank=True,
+                            to='random.Model',
+                            null=True,
+                        ),
+                    )
+                ]
+        """
+        after = """
+            from django.db import models
+
+
+            class MyModel(models.Model):
+                operations = [
+                    migrations.AddField(
+                        field=models.ForeignKey(
+                            related_name='model',
+                            blank=True,
+                            to='random.Model',
+                            null=True,
+                        on_delete = models.CASCADE),
+                    )
+                ]
+        """
+        self.assertCodemod(before, after)
+
+    def test_positional_args_without_on_delete(self) -> None:
+        before = """
+            from django.db import models
+
+            models.OneToOneField('model')
+        """
+        after = """
+            from django.db import models
+
+            models.OneToOneField('model', on_delete = models.CASCADE)
+        """
+        self.assertCodemod(before, after)
+
+    def test_positional_args_with_on_delete(self) -> None:
+        # shouldn't do anything
```

Comment:
Actually for the ones where nothing is expected to change, a docstring would be welcome.